ktor-server/jvm/src/io/ktor/server/application/plugins/api/ServerPluginFactory.kt

```diff
@@ -11,9 +11,24 @@ import io.ktor.util.pipeline.*
  * A factory class to be passed to the [install] function for creating a [ServerPlugin]
  * instance and installing it into the current application context.
  **/
-public abstract class ServerPluginFactory<Configuration : Any>(
-    public val name: String
-) : ApplicationPlugin<ApplicationCallPipeline, Configuration, ServerPlugin<Configuration>>
+public sealed class ServerPluginFactory<Configuration : Any, TPlugin : ServerPlugin<Configuration>> :
```

Comment:
Maybe add some information about the second parameter to KDoc?

Reply:
No need, I removed the second parameter =)